### ISSUE TITLE
Hadoop jobs fail with HTTP 500 error if Hive/Pig homes are not set in properties file

### ISSUE THREAD

**Issue by sriramkrishnan** [Netflix/genie#11]
The entire stack trace is:
java.lang.NullPointerException
    at java.lang.ProcessEnvironment.validateValue(ProcessEnvironment.java:102)
    at java.lang.ProcessEnvironment.access$400(ProcessEnvironment.java:44)
    at java.lang.ProcessEnvironment$Value.valueOf(ProcessEnvironment.java:185)
    at java.lang.ProcessEnvironment$StringEnvironment.put(ProcessEnvironment.java:224)
    at java.lang.ProcessEnvironment$StringEnvironment.put(ProcessEnvironment.java:203)
    at java.util.AbstractMap.putAll(AbstractMap.java:256)
    at com.netflix.genie.server.jobmanager.impl.HadoopJobManager.launch(HadoopJobManager.java:168)
    at com.netflix.genie.server.services.impl.GenieExecutionServiceImpl.submitJob(GenieExecutionServiceImpl.java:210)
    at com.netflix.genie.server.resources.JobResourceV0.submitJob(JobResourceV0.java:123)

Basically, the put fails on the map if the values are null. Hive/Pig homes should be optional  - only required if Genie is set up for Hive or Pig.
